docs/standard/library-guidance/sourcelink.md

```diff
@@ -29,9 +29,9 @@ You can use [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuG
 > * <xref:System.Diagnostics.DebuggerStepThroughAttribute> instructs the debugger to step through the code instead of stepping into the code.
 > * <xref:System.Diagnostics.DebuggerBrowsableAttribute> controls whether a member is displayed in the debugger variable windows.
 
-**✔️ CONSIDER** including symbol files (`*.pdb`) in the NuGet package.
+**✔️ CONSIDER** publishing symbol files (`*.pdb`).
```

Comment:
would it be helpful to say "with your package"?

Reply:
I like the generic wording that @JamesNK used here. That way this page doesn't have to be updated again should the recommendation on the "NuGet" page change in the future.